aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -195,9 +303,13 @@ Resources:
       AutoScalingGroupName: {Ref: WebServer}
       MinSize: 2
       Recurrence: "0 3 * * TUE-SAT"
+<%   end -%>
```

Comment:
Re: all of the above MinSize changes, these will have to be disabled for HOC, where we're going to be running much larger, correct?

Is there a clean way to make sure that manual changes override these schedules without having to deploy code?